tests/vmi_multus_test.go

```diff
@@ -580,6 +580,36 @@ var _ = Describe("SRIOV", func() {
 
 	tests.BeforeAll(func() {
 		tests.BeforeTestCleanup()
+		// Check if the hardware supports SRIOV
```

Comment:
This should be in a helper function to make it more readable. It is a lot of code occupying BeforeAll.